api.test_permission.PermissionTestCase - new permissions + types added

`tests.foreman.api.test_permission.PermissionTestCase.test_positive_search`
`tests.foreman.api.test_permission.PermissionTestCase.test_positive_search_by_resource_type`

```
AssertionError: {
  "added_names": [
    "import_templates",
    "export_templates"
  ],
  "added_resource_types": [   "Template"  ] }
```
```
AssertionError: {  "Host": {   "added": [    "play_roles_on_host"   ]  },  "HostClass": {   "added": [],   "removed": [    "play_roles_on_host"   ]  } }
```

--- a/robottelo/constants.py
+++ b/robottelo/constants.py
@@ -756,7 +756,6 @@ PERMISSIONS = {
     ],
     'HostClass': [
         'edit_classes',
-        'play_roles_on_host',
     ],
     'Hostgroup': [
         'view_hostgroups',
@@ -934,6 +933,7 @@ PERMISSIONS = {
         'edit_discovered_hosts',
         'edit_hosts',
         'ipmi_boot_hosts',
+        'play_roles_on_host',
         'power_hosts',
         'provision_discovered_hosts',
         'puppetrun_hosts',

